server/jms/src/main/java/org/hawkular/apm/server/jms/CommunicationDetailsCacheMDB.java

```diff
@@ -30,17 +30,20 @@
 import com.fasterxml.jackson.core.type.TypeReference;
 
 /**
+ * This MDB is provided to populate the CommunicationDetailsCache when Communication Details.
+ * Each clustered APM server node will receive the communication details data, so each cache
+ * is expected to only be a local cache.
+ *
+ * Discussion regarding potential future use of a distributed cache is associated with HWKAPM-479.
+ * If this change happens, then this MDB should be converted back to a durable (load balanced) subscriber.
+ *
  * @author gbrown
  */
-@MessageDriven(name = "CommunicationDetails_Cache", messageListenerInterface = MessageListener.class,
+@MessageDriven(name = "CommunicationDetails_CommunicationDetailsCache", messageListenerInterface = MessageListener.class,
         activationConfig =
         {
                 @ActivationConfigProperty(propertyName = "destinationType", propertyValue = "javax.jms.Topic"),
-                @ActivationConfigProperty(propertyName = "destination", propertyValue = "CommunicationDetails"),
-                @ActivationConfigProperty(propertyName = "subscriptionDurability", propertyValue = "Durable"),
```

Comment:
I think what you write matches what I recall from the past. Having different client ids (e.g. uuid of the WF10 server you are running on) + durable subscription sounds like what we need.

How do you bootstrap the cache?